tools/bin/commands/util/openshift_funcs

```diff
@@ -222,3 +222,42 @@ guess_route() {
     fi
     echo "${project}.${domain#*.}"
 }
+
+create_api_cookie() {
+
+    local COOKIE1="$HOME/syndesis_cookie.txt"
+    # second cookie needed because curl flushes old values upon redirect. Might be a bug or some obscure flag missing
+    local COOKIE2="$(mktemp /tmp/syndesis_cookie.XXX.txt)"
```

Comment:
could we use lowercase vars for local variable and resever uppercase vars for exported env vars ? (its just that the other commands do it that way)